src/Identifier/OrmIdentifier.php

```diff
@@ -46,10 +48,30 @@ class OrmIdentifier extends AbstractIdentifier
         ],
         'userModel' => 'Users',
         'finder' => 'all',
-        'passwordHasher' => DefaultPasswordHasher::class
+        'passwordHasher' => null
     ];
 
     /**
+     * Return password hasher object.
+     *
+     * @return \Authentication\PasswordHasher\PasswordHasherInterface Password hasher instance.
+     */
+    public function getPasswordHasher()
+    {
+        if ($this->_passwordHasher === null) {
+            $passwordHasher = $this->getConfig('passwordHasher');
+            if ($passwordHasher !== null) {
+                $passwordHasher = PasswordHasherFactory::build($passwordHasher);
+            } else {
+                $passwordHasher = $this->traitGetPasswordHasher();
```

Comment:
I've been told to remove it as well but I really would prefer to keep it except somebody tells me how I get phpstorm to visually indicate the visibility for me...

PSR my ass. 💩